packages/office-ui-fabric-react/src/components/ProgressIndicator/ProgressIndicator.base.tsx

```diff
@@ -93,23 +107,42 @@ export class ProgressIndicatorBase extends BaseComponent<IProgressIndicatorProps
       transition: percentComplete !== undefined && percentComplete < ZERO_THRESHOLD ? 'none' : undefined
     };
 
-    const ariaValueMin = percentComplete !== undefined ? 0 : undefined;
-    const ariaValueMax = percentComplete !== undefined ? 100 : undefined;
-    const ariaValueNow = percentComplete !== undefined ? Math.floor(percentComplete!) : undefined;
+    const percentCompleteAriaText = ariaValueText ? ariaValueText : this.state.percentComplete + '%';
+    const indeterminateAriaText = ariaValueText ? ariaValueText : 'Working on it';
 
     return (
       <div className={classNames.itemProgress}>
         <div className={classNames.progressTrack} />
-        <div
-          className={classNames.progressBar}
-          style={progressBarStyles}
-          role="progressbar"
-          aria-valuemin={ariaValueMin}
-          aria-valuemax={ariaValueMax}
-          aria-valuenow={ariaValueNow}
-          aria-valuetext={ariaValueText}
-        />
+        <div className={classNames.progressBar} style={progressBarStyles} />
+        {percentComplete !== undefined ? (
+          // aria-live is used in lieu of aria-valuenow/min/max and role='progressbar to
+          // circumvent the need for tabbing to hear the screen reader
+          <span aria-live="polite" className={classNames.ariaText}>
```

Comment:
hmm. If you have 20 progress indicators in the ui, what will happen?

Should this be opt in?

Reply:
@betrue-final-final what do you think of the opt in option?

Reply:
Too many built-in accessibility hooks routinely make a UX *inaccessible*. The host application must decide how it will present status in a consistent manner to the user. If we bake in an `aria-live` region, we will likely mess up a parent element which is just trying to use this control for its visuals.
`ProgressIndicator` isn't a 'smart' component; it shouldn't be in the business of trying to do more than just display and represent the data. the `progressbar` role and `aria-labels` (where appropriate) should be enough.
If the host application cares, *it* should wrap the `ProgressIndicator` in a live-region.

Reply:
It's clear that we need to work out how we are implementing screen reader, although I think @betrue-final-final can back me up here that screen reading functionality does not make this a smart component.